pipeline.cpp

```diff
@@ -80,7 +81,42 @@ void DrawLine(Buffer2D<PIXEL> & target, Vertex* const triangle, Attributes* cons
  ************************************************************/
 void DrawTriangle(Buffer2D<PIXEL> & target, Vertex* const triangle, Attributes* const attrs, Attributes* const uniforms, FragmentShader* const frag)
 {
-    // Your code goes here
+    // Find a bounding box for the triangle
```

Comment:
He provided a MIN3 and MAX3 for us to be able to use. I started doing the same thing here you did before I saw that.